foodhub.sql

```diff
@@ -0,0 +1,137 @@
+# ************************************************************
+# Sequel Ace SQL dump
+# Version 20070
+#
+# https://sequel-ace.com/
+# https://github.com/Sequel-Ace/Sequel-Ace
+#
+# Host: localhost (MySQL 11.5.2-MariaDB-ubu2404)
+# Database: foodhub
+# Generation Time: 2024-12-09 15:12:22 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+SET NAMES utf8mb4;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE='NO_AUTO_VALUE_ON_ZERO', SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table ingredients
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `ingredients`;
+
+CREATE TABLE `ingredients` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email_id` int(11) NOT NULL,
```

Comment:
Isn't `user_id` better considering the table is called users and we could decide to add more fields to a 'user' later?

More attention to detail stuff...